app/src/components/intelligence/memory-workspace.css

```
@@ -0,0 +1,613 @@
+  letter-spacing: -0.01em;
+  color: var(--ink);
+  margin: 0 0 16px;
+  line-height: 1.35;
+}
+
+.mw-letter-body {
+  font-size: 15px;
+  line-height: 1.7;
+  color: var(--ink);
+  white-space: pre-wrap;
+  word-wrap: break-word;
```

Stylelint (17.10.0)

[error] Expected "word-wrap" to be "overflow-wrap" (property-no-deprecated)

(property-no-deprecated)